src/lib/theme/index.tsx

```diff
@@ -1,4 +1,4 @@
-import 'lib/assets/fonts/index.css'
+import '../assets/fonts/index.css' // microbundle requires relative css paths
```

Comment:
ew :(

Reply:
microbundle has been a let down, but I think transitioning would be a huge time sink, and there's better things we could be spending time on. See `yarn assets:generate`.